### PR TITLE
[4] Empty state for com_redirect

### PR DESCRIPTION
**Pull request by PhilETaylor**
### Summary of Changes

Slightly different this time as this one needs a NEW and a BULK IMPORT action in the toolbar, the latter requiring a modal

Screenshot 2021-04-27 at 21 01 31

### Testing Instructions

Also test that bulk import still works please. (It "Works on my computer")

### CODE REVIEW

File: administrator/language/en-GB/com_redirect.ini [hunk 1]
```diff
@@ -22,6 +22,9 @@ COM_REDIRECT_DEFAULT_IMPORT_STATE_LABEL="Import State"
 COM_REDIRECT_DISABLE_LINK="Disable Link"
 COM_REDIRECT_EDIT_LINK="Edit Link #%d"
 COM_REDIRECT_EDIT_PLUGIN_SETTINGS="Edit Plugin Settings"
+COM_REDIRECT_EMPTYSTATE_TITLE="Redirects: Links"
+COM_REDIRECT_EMPTYSTATE_CONTENT="This component is used for redirecting URLs for web pages that no longer exist on your website to web pages that are working."
+COM_REDIRECT_EMPTYSTATE_BUTTON_ADD="Add new Link"
```
Comment (Quy) [inline administrator/language/en-GB/com_redirect.ini]:
```suggestion
COM_REDIRECT_EMPTYSTATE_BUTTON_ADD="Add new link"
COM_REDIRECT_EMPTYSTATE_CONTENT="This component is used for redirecting URLs for web pages that no longer exist on your website to web pages that are working."
COM_REDIRECT_EMPTYSTATE_TITLE="Redirects: Links"
```

File: administrator/language/en-GB/com_redirect.ini [hunk 2]
```diff
@@ -22,6 +22,9 @@ COM_REDIRECT_DEFAULT_IMPORT_STATE_LABEL="Import State"
 COM_REDIRECT_DISABLE_LINK="Disable Link"
 COM_REDIRECT_EDIT_LINK="Edit Link #%d"
 COM_REDIRECT_EDIT_PLUGIN_SETTINGS="Edit Plugin Settings"
+COM_REDIRECT_EMPTYSTATE_BUTTON_ADD="Add new link"
+COM_REDIRECT_EMPTYSTATE_CONTENT="This component is used for redirecting URLs for web pages that no longer exist on your website to web pages that are working."
```
Comment (Quy) [inline administrator/language/en-GB/com_redirect.ini]:
```suggestion
COM_REDIRECT_EMPTYSTATE_CONTENT="Redirect URLs for web pages that no longer exist on your website to web pages that are working."
```
Simplify?

Comment (PhilETaylor) [reply administrator/language/en-GB/com_redirect.ini]: I just copied and pasted from docs.Joomla.org haha

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9d2f350c1e79036e0b9127c46e8dbf9a779ddb88<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33384">issues.joomla.org/tracker/joomla-cms/33384</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 9d2f350c1e79036e0b9127c46e8dbf9a779ddb88<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33384">issues.joomla.org/tracker/joomla-cms/33384</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33384">issues.joomla.org/tracker/joomla-cms/33384</a>.</sub>

File: administrator/components/com_redirect/tmpl/links/emptystate.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_checkin
```
Comment (brianteeman) [inline administrator/components/com_redirect/tmpl/links/emptystate.php]:
```suggestion
 * @subpackage  com_redirect
```

**Review by brianteeman — CHANGES_REQUESTED** [review]

**Comment by drmenzelit** [comment]
Thanks

**Pull request merged**